runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -108,18 +109,16 @@ pub fn measure_actions(
 }
 
 // TODO: super-ugly, can achieve the same via higher-level wrappers over POSIX read().
```

Comment:
It seems that this TODO is getting addressed. @olonho Can you review this implementation? (according to git, it was your TODO)

Reply:
He's OOO today, but I've spoken to him about this solution and he has ideas on how to improve it.